packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+
+### Check your rate limits with Okta Rate Limit headers
+
+The Rate Limit headers returned when client-based rate limiting is enabled are very similar to the headers returned today through the [org-wide rate limits](/docs/reference/rl-best-practices/). The difference is that the header value is specific to a given client/IP/device combination rather than the org-wide Rate Limit values. Okta provides three headers in each response to report client-specific rate limits.
+
+> **Note:** If client-based rate limiting is in **Log per client** or **No action** mode, headers that are returned still reflect the org-wide rate limits.
+
+For client-specific rate limits, three headers show the limit that is being enforced, when it resets, and how close you are to hitting the limit:
+
+* `X-Rate-Limit-Limit` - The rate limit ceiling that is applicable for the current request to the specific client/IP/device identifier combination
+* `X-Rate-Limit-Remaining` - The number of requests left for the specific client/IP/device identifier combination during the current rate-limit window
+* `X-Rate-Limit-Reset` - The time when the rate limit resets, specified in [UTC epoch time](https://www.epochconverter.com/) for the specific client/IP/device identifier combination
+
+For example:
+
+```
+HTTP/1.1 200 OK
+X-Rate-Limit-Limit: 60
+X-Rate-Limit-Remaining: 35
+X-Rate-Limit-Reset: 1516307596
+```
+
+When a specific client/IP/device identifier combination exceeds either the per minute request limit (60 requests per minute) or the concurrent limit (two concurrent requests), the `/authorize` request returns an HTTP 429 error.
+
+### How to enable this feature
+
+> **Note:** For all orgs created after September 2020, this feature is automatically set to **Enforce and log per client (recommended)** mode.
+
+For existing orgs, to configure client-based rate limiting:
+
+> **Note:** The Admin Console is required for these steps. If you see **Developer Console** in the top left of the page, click it and select **Classic UI** to switch.
+
+1. From the Admin Console select **Settings** > **Account**, and then scroll down to the **Client-based rate limiting** section.
+
+2. Select the type of **Rate limit per client** that you want to implement:
+
+    * Select **Log and enforce client (recommended)** to enable client-based rate limiting.
+
+    * Select **Log per client** to enable client-based rate limiting in preview mode.
+
+    * Select **No action** to disable client-based rate limiting.
+
+### Frequently asked questions
+
+**Question:** Which endpoints are covered under client-based rate limiting?<br>
+**Answer:** Today, client-based rate limiting only applies to an authorization server's `/authorize` endpoint.
+
+**Question:** How is the client-specific rate limit determined?<br>
```

Comment:
Remove &lt;br&gt; tagging throughout and use the bolded question followed by a new paragraph for the answer.

Reply:
And didn't we just say above that the limit was 60 requests per minute?